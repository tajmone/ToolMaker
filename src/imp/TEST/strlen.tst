%%SET A(ASDFqw)
%%SET B(JKLOqw)
%%(IMP$STRLEN("%%(A)%%(B)"))
%%(IMP$STRLEN("A"))
%%IF((IMP$STRLEN("A")) = 1)
%%("A")
%%ELSE
%%FEL!
%%ENDIF
%%((IMP$STRLEN("ABC")) + 2)
%%IF((IMP$STRLEN(A)) = 6)
%%(A)
%%ELSE
%%FEL!
%%ENDIF
