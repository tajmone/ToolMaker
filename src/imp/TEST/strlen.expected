########## strlen ##########
%%SET A(ASDFqw)
%%SET B(JKLOqw)
%%(IMP$STRLEN("%%(A)%%(B)"))
%%(IMP$STRLEN("A"))
%%IF((IMP$STRLEN("A")) = 1)
%%("A")
%%ELSE
%%FEL!
%%ENDIF
%%((IMP$STRLEN("ABC")) + 2)
%%IF((IMP$STRLEN(A)) = 6)
%%(A)
%%ELSE
%%FEL!
%%ENDIF
########## ../imp strlen.tst >strlen.tst.tmp; cat strlen.tst.tmp; rm strlen.tst.tmp ##########
12
1
A
5
ASDFqw
